source/includes/masterportal/_applications.md

```diff
@@ -0,0 +1,92 @@
+### Applications
+
+Application's login can be automated by MasterPortal. It has an entrypoint that points to the MasterPortal proxy and can have credentials added to it. 
+
+#### Add Credentials
+
+```shell
+Example Request: 
+curl --request POST \
+  --url 'https://cloudmc_endpoint/rest/services/mp-devel/test-applications/apps/b6c4ca5b-e85a-41d3-80b2-5d21d6ce7060?operation=addCredentials' \
```

Comment:
The url used in the API doc example should be pointing to the `/v2` context. Like in other doc examples.
_ex:_ `https://cloudmc_endpoint/v2/users/:id/additional_roles`